usaspending_api/references/serializers.py

```diff
@@ -73,36 +75,19 @@ class Meta:
         }
 
 
-class ObjectClassSerializer(LimitableSerializer):
-
-    class Meta:
-        model = ObjectClass
-        fields = '__all__'
-
-
-class RefProgramActivitySerializer(LimitableSerializer):
-
-    class Meta:
-        model = RefProgramActivity
-        fields = '__all__'
-
-
-class RefProgramActivityBriefSerializer(serializers.ModelSerializer):
-    prefetchable = False
-    code = serializers.CharField(source='program_activity_code')
-    title = serializers.CharField(source='program_activity_name')
+class ProgramActivitySerializer(LimitableSerializer):
 
     class Meta:
 
         model = RefProgramActivity
-        fields = ('id', 'code', 'title')
+        fields = ('id', 'program_activity_code', 'program_activity_name')
 
 
-class ObjectClassBriefSerializer(serializers.ModelSerializer):
-    prefetchable = False
-    major_object_class_code = serializers.CharField(source='major_object_class')
+class ObjectClassSerializer(LimitableSerializer):
 
     class Meta:
 
         model = ObjectClass
-        fields = ('major_object_class_code', 'major_object_class_name', )
+        fields = (
```

Comment:
This will now mean that users can't acquire any of the fields other than the ones expressly denoted here. Is this what we want?